src/app/app.module.ts

```diff
@@ -16,7 +16,7 @@ import { SyncStatusModule } from './sync-status/sync-status.module';
 import { NavigationModule } from './navigation/navigation.module';
 import { LatestChangesModule } from './latest-changes/latest-changes.module';
 import { DashboardModule } from './dashboard/dashboard.module';
-
+//import { UserAccountComponent } from './user/user-account/user-account.component';
```

Comment:
as it's commented out, it is not needed. So let's remove this before merging it into the master branch